content/install/server/database.md

```diff
@@ -22,8 +22,8 @@ services:
   drone-server:
     image: drone/drone:{{% version %}}
     environment:
-+     DRONE_DATABASE_DRIVER: mysql
-+     DRONE_DATABASE_DATASOURCE: root:password@tcp(1.2.3.4:3306)/drone?parseTime=true
++     DRONE_DATABASE_DRIVER:mysql
```

Comment:
`- DRONE_DATABASE_DRIVER=mysql` this array syntax with equal sign as separator works best.

Reply:
Is it supposed to work the same for both `mysql` and `postgres`? because when I switch any of the 2 lines from `:` to `=`for my PostgreSQL instance, it just shows the never ending loop of the following error message.

```log
drone-agent_1   | INFO: <redacted_date> <redacted_time> grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: Error while dialing dial tcp <redacted_ip>:<redacted_port>: getsockopt: connection refused"; Reconnecting to {drone-server:<redacted_port> <nil>}
drone-agent_1   | <redacted_date> <redacted_time> grpc error: done(): code: Unavailable: rpc error: code = Unavailable desc = grpc: the connection is unavailable
```

 Switching them back to `:` fixes the problem.